### PR TITLE
Add more images button

### PR DESCRIPTION
**Pull request by rd-code-23**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "start": "npm run lint & node ./server.js",
     "dev": "npm run lint & nodemon ./server.js",
     "lint": "eslint . --ext .js"
+
```
Comment (furkanmutlu) [inline backend/package.json:10]: please remove the extra space here.

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -37,22 +41,61 @@ function App() {
...
+    }
+
+    const receivedImages = await getImages(imageType, limit, offset);
+
+    setImages([...images, ...receivedImages]);
+    updateOffsetValue();
+  }
+
+  const setInitialOffsetValue = (value) => {
+    if (value === 'gifs') {
```
Comment (furkanmutlu) [inline frontend/src/App.jsx:84]: Can we use ternary operator here? I think it is more readable that way. What do you think ?

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -37,22 +41,61 @@ function App() {
...
+      setOffset(0);
+    } else {
+      setOffset(1);
+    }
+  }
+
+  const updateOffsetValue = () => {
+    if (imageType === 'gifs') {
```
Comment (furkanmutlu) [inline frontend/src/App.jsx:92]: Can we use ternary operator here? I think it is more readable that way. What do you think ?

File: backend/routes/gifs.js [hunk 4]
```diff
@@ -3,18 +3,20 @@ const { getTrendingData, getSearchedImages } = require('../services/API');
...
     }
 });
 
 router.route('/search').get(async (req, res) => {
-    const searchTerm = req.query.searchTerm;
-
     try {
-        res.json(await getSearchedImages('gif', 25, searchTerm));
+        const { searchTerm, limit, offset } = req.query;
+
```
Comment (furkanmutlu) [inline backend/routes/gifs.js:18]: Could you please remove the trailing spaces here / or remove eslint no-trailing-spaces rule. Looks like eslint throws error while building the app.

**Review by furkanmutlu — CHANGES_REQUESTED** [review]
Overall great work! Just checked it on my local machine, I like the ui and your coding style. I have tiny comments for you to fix. Please let me know if you have any questions. Thanks again :) looking forward to do more projects with you.

File: backend/package.json (hunk 1 above)
Comment (rd-code-23) [reply backend/package.json:10]: @furkanmutlu 
 Its good you went thru my code, I appreciate the feedback. Will do these changes in the next few days.

Comment (rd-code-23) [reply backend/package.json]: @furkanmutlu 
I made the changes, have another look

Comment (furkanmutlu) [reply backend/package.json]: did you push your changes?

Comment (rd-code-23) [reply backend/package.json]: I pushed it and it does show up on my branch, but it didn't update with the pull request for some reason.
I will try to figure it out today.

Comment (rd-code-23) [reply backend/package.json]: Yes, iIdid push, you can see the changes in the commits tab called 'refactor'

Comment (rd-code-23) [reply backend/package.json]: https://github.com/furkanmutlu/fullstack-image-app/pull/7/commits/a820500afddc372e82257ec123d0a4c9b9acd858

Comment (rd-code-23) [reply backend/package.json]: @furkanmutlu 
did you get my last message?

**Review by furkanmutlu — APPROVED** [review]
hi, sorry for the very late response, I just had time to review your PR. It looks great, thanks for the contribution !

**Pull request merged**